rust/sbd-server/tests/suite.rs

```diff
@@ -0,0 +1,27 @@
+#[test]
+fn suite() {
```

Comment:
totally dependent on system resources, on my system it's around 30 seconds after deleting target, but caching obviously makes this take no time at all... We could add a job ahead of the test job that causes them to be built with the correct settings, but I don't know how easy it would be to make sure the settings stay in sync. Feels pretty low priority to me.